test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -100,8 +106,12 @@ void static data_init(perf_metrics_t * data) {
     data->warmup = WARMUP; /*number of initial iterations to skip*/
     data->unit = MB;
     data->validate = false;
+    data->target_data = false;
     data->my_node = shmem_my_pe();
     data->num_pes = shmem_n_pes();
+    data->midpt = data->num_pes/2;
```

Comment:
I catch launching an odd number of PEs as invalid usage and will throw an error and exit